model_analyzer/entrypoint.py

```diff
@@ -92,6 +93,11 @@ def get_server_handle(config, gpus):
             ' Model Analyzer does not have access to the model repository of'
             ' the remote Triton Server.')
     elif config.triton_launch_mode == 'local':
+        path = config.get_config()['triton_server_path'].value()
```

Comment:
I know all of the existing code desperately needs a refactor for cleanliness, but lets make sure the new code is clean. Put all of this added code into a function called validate_triton_server_path()

Reply:
Added `validate_triton_server_path()`